README.md

```diff
@@ -35,30 +35,40 @@ Screenshot (will need to be updated as syntax coloring and UI polish is added):
 
 ![xi screenshot](/doc/img/xi-mac-screenshot.png?raw=true)
 
-
 ## Getting started
 
 ### Requirements
+
 - [Xcode 9.x](https://developer.apple.com/xcode/)
-- [Rust](https://www.rust-lang.org/). We test against the latest stable version,
-and recommend installing through [rustup](https://rustup.rs).
+- [Rust](https://www.rust-lang.org/). We test against the latest stable version, and recommend installing through [rustup](https://rustup.rs).
```

Comment:
total nit, but we manually wrap markdown at either 72 or 80 columns. This makes it render correctly as plaintext, and markdown ignores single newlines when rendering as html.

Reply:
Reverting that now